<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="sw_1">
       <widget class="QLabel" name="lbl_welcome">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>90</y>
          <width>651</width>
          <height>291</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;Welcome to&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;EDG Compiled Makers&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_1">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>360</y>
          <width>211</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Search for Makers</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_settings">
        <property name="geometry">
         <rect>
          <x>810</x>
          <y>530</y>
          <width>75</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Settings</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="sw_2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QFrame" name="body_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QFrame" name="frm_buttons">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="btn_refresh">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Refresh List</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_update">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Update Maker</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_add">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Add Maker</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_delete">
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>Delete Maker</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frm_table">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QTableWidget" name="tbl_1">
                <property name="editTriggers">
                 <set>QAbstractItemView::NoEditTriggers</set>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::SingleSelection</enum>
                </property>
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectRows</enum>
                </property>
                <property name="sortingEnabled">
                 <bool>false</bool>
                </property>
                <column>
                 <property name="text">
                  <string>ID</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>MAKER</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>CATEGORY</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>MODEL NO</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>DESCRIPTION</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>REMARKS</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>SYMBOL</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>LINK</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>NOTES</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="sw_3">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QFrame" name="body_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QFrame" name="frame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="lbl_1">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Database Connection Settings&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_back">
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>Back</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="lbl_2">
                <property name="text">
                 <string>Database Folder Path</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTextEdit" name="txt_db_path">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_sel_db_path">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>change</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="sw_4">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>110</y>
             <width>131</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>NEW CATEGORY NAME</string>
           </property>
          </widget>
          <widget class="QTextEdit" name="textEdit">
           <property name="geometry">
            <rect>
             <x>320</x>
             <y>110</y>
             <width>241</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="btn_add_cat">
           <property name="geometry">
            <rect>
             <x>580</x>
             <y>110</y>
             <width>51</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>add</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btn_save_cat">
           <property name="geometry">
            <rect>
             <x>780</x>
             <y>510</y>
             <width>75</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
          <widget class="QListWidget" name="listWidget">
           <property name="geometry">
            <rect>
             <x>320</x>
             <y>160</y>
             <width>241</width>
             <height>291</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton">
           <property name="geometry">
            <rect>
             <x>320</x>
             <y>460</y>
             <width>101</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Delete Category</string>
           </property>
          </widget>
          <widget class="QPushButton" name="btn_cancel_cat">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>510</y>
             <width>75</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="sw_5">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QFrame" name="frame_4">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QFrame" name="frame_6">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLabel" name="lbl_4">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Add New Maker / Update Maker&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_5">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <widget class="QLabel" name="label_2">
              <property name="geometry">
               <rect>
                <x>210</x>
                <y>100</y>
                <width>101</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>MODEL NO</string>
              </property>
             </widget>
             <widget class="QTextEdit" name="txt_model_no">
              <property name="geometry">
               <rect>
                <x>340</x>
                <y>90</y>
                <width>241</width>
                <height>31</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="label_3">
              <property name="geometry">
               <rect>
                <x>210</x>
                <y>140</y>
                <width>101</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>CATEGORY</string>
              </property>
             </widget>
             <widget class="QTextEdit" name="txt_desc">
              <property name="geometry">
               <rect>
                <x>340</x>
                <y>170</y>
                <width>241</width>
                <height>31</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="label_4">
              <property name="geometry">
               <rect>
                <x>210</x>
                <y>180</y>
                <width>101</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>DESCRIPTION</string>
              </property>
             </widget>
             <widget class="QComboBox" name="cbo_cat">
              <property name="geometry">
               <rect>
                <x>340</x>
                <y>130</y>
                <width>241</width>
                <height>31</height>
               </rect>
              </property>
             </widget>
             <widget class="QTextEdit" name="txt_remarks">
              <property name="geometry">
               <rect>
                <x>340</x>
                <y>210</y>
                <width>241</width>
                <height>31</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="label_5">
              <property name="geometry">
               <rect>
                <x>210</x>
                <y>220</y>
                <width>101</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>REMARKS</string>
              </property>
             </widget>
             <widget class="QTextEdit" name="txt_maker_link">
              <property name="geometry">
               <rect>
                <x>340</x>
                <y>250</y>
                <width>241</width>
                <height>31</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="label_6">
              <property name="geometry">
               <rect>
                <x>210</x>
                <y>260</y>
                <width>101</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>MAKER FILE LINK</string>
              </property>
             </widget>
             <widget class="QTextEdit" name="txt_symbol_link">
              <property name="geometry">
               <rect>
                <x>340</x>
                <y>290</y>
                <width>241</width>
                <height>31</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="label_7">
              <property name="geometry">
               <rect>
                <x>210</x>
                <y>300</y>
                <width>101</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>SYMBOL LINK</string>
              </property>
             </widget>
             <widget class="QPushButton" name="btn_attach_file">
              <property name="geometry">
               <rect>
                <x>590</x>
                <y>250</y>
                <width>51</width>
                <height>31</height>
               </rect>
              </property>
              <property name="text">
               <string>attach</string>
              </property>
             </widget>
             <widget class="QPushButton" name="btn_attach_symbol">
              <property name="geometry">
               <rect>
                <x>590</x>
                <y>290</y>
                <width>51</width>
                <height>31</height>
               </rect>
              </property>
              <property name="text">
               <string>attach</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_8">
              <property name="geometry">
               <rect>
                <x>210</x>
                <y>340</y>
                <width>101</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>NOTES</string>
              </property>
             </widget>
             <widget class="QTextEdit" name="txt_notes">
              <property name="geometry">
               <rect>
                <x>340</x>
                <y>330</y>
                <width>241</width>
                <height>101</height>
               </rect>
              </property>
             </widget>
             <widget class="QPushButton" name="btn_add_category">
              <property name="geometry">
               <rect>
                <x>590</x>
                <y>130</y>
                <width>51</width>
                <height>31</height>
               </rect>
              </property>
              <property name="text">
               <string>add</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_16">
              <property name="geometry">
               <rect>
                <x>210</x>
                <y>60</y>
                <width>101</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>MAKER</string>
              </property>
             </widget>
             <widget class="QTextEdit" name="txt_maker">
              <property name="geometry">
               <rect>
                <x>340</x>
                <y>50</y>
                <width>241</width>
                <height>31</height>
               </rect>
              </property>
             </widget>
             <widget class="QPushButton" name="btn_cancel_add">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>440</y>
                <width>75</width>
                <height>31</height>
               </rect>
              </property>
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
             <widget class="QPushButton" name="btn_save_add">
              <property name="geometry">
               <rect>
                <x>770</x>
                <y>440</y>
                <width>75</width>
                <height>31</height>
               </rect>
              </property>
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
             <widget class="QTextEdit" name="txt_id">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="geometry">
               <rect>
                <x>340</x>
                <y>10</y>
                <width>241</width>
                <height>31</height>
               </rect>
              </property>
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Auto generated&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_17">
              <property name="geometry">
               <rect>
                <x>210</x>
                <y>20</y>
                <width>101</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>ID</string>
              </property>
             </widget>
             <widget class="QPushButton" name="btn_save_update">
              <property name="geometry">
               <rect>
                <x>780</x>
                <y>440</y>
                <width>75</width>
                <height>31</height>
               </rect>
              </property>
              <property name="text">
               <string>Update</string>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
